ospfd/ospf_packet.c

```diff
@@ -891,6 +891,17 @@ ospf_hello (struct ip *iph, struct ospf_header *ospfh,
   p.prefixlen = ip_masklen (hello->network_mask);
   p.u.prefix4 = iph->ip_src;
 
+  if (IS_DEBUG_OSPF_TRACE)
```

Comment:
he purpose of the log is for daemon internal activity and provides different filter level of logs. If any objection will remove it.